napari/components/viewer_model.py

```diff
@@ -94,8 +94,28 @@ def __init__(
         self.mouse_move_callbacks = []
         # Hold callbacks for when mouse is pressed, dragged, and released
         self.mouse_drag_callbacks = []
+        # Hold callbacks for when mouse wheel is scrolled
+
+        def _scroll(viewer, event):
+            from .._viewer_key_bindings import (
```

Comment:
This import breaks at top level as a circular import, so we should pull out increment_dims_left and increment_dims_right. I think it would work well as a dims method but I think @jni had some bones to pick with that?

Reply:
I didn't have specific bones, I just wasn't sure. =) I think I was worried about the fact that dims will get refactored soon, but this will be affected either way. Thinking about it now, I agree, should be a Dims method with an axis kwarg and a None default, in which case last axis is used.

Reply:
So I tried pulling this out but I don't think it's that simple? If I put `increment_dims` on Dims, then we don't have access to `last_used`, because it belongs to `qt_dims`. If we put it in `qt_dims`, then we can't access it in `viewer_model` for the `_scroll` method. 

Also, regardless of which we put it in, we still need the `increment_dims_left/right` in the key bindings, in order to bind left and right arrows, so it would just call either `qt_dims`' or `dims`' method since it has access to both.

So now I'm not too sure where to put it. Left and right arrows works with it pretty much anywhere, but the scroll not so much.

Reply:
> So now I'm not too sure where to put it. Left and right arrows works with it pretty much anywhere, but the scroll not so much.

For now we should probably put `last_used` on dims but we might want to think about another place longer term!